src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/TypeCast.cs

```diff
@@ -851,6 +988,8 @@ public static unsafe object IsInstanceOf(MethodTable* pTargetType, object obj)
             // @TODO: consider using the cache directly, but beware of IDynamicInterfaceCastable in the interface case
```

Comment:
Is this TODO still actionable? Even if actionable, perhaps better to log an issue instead.

Reply:
I think it is - but it would require perf measurements. We have https://github.com/dotnet/runtime/issues/84464 tracking casting perf, I don't think it should be resolved with this. This PR specifically only touches the cast to class/interface cases because that's what I saw in the profile. It doesn't try to do anything for this slow(er) helper - I only had to touch the slow helper because it calls into the fast helpers and those no longer handle variance after this PR.

Reply:
I see. Makes sense.